index.html

```diff
@@ -30,16 +42,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box" style="background-color:teal;">Box 1</div>
+                <div class="box" style="background-color:gold;">Box 2</div>
```

Comment:
Great job using inline styling to set the colors of the box, however, mixing inline styles with external styling libraries can get messy. Instead of storing your styles in one place, now you have 2.

One of the main principles of programming is writing code that is reusable. In this case, for each of the boxes you want all the boxes the same size, but you want them different colors. One of the ways you could achieve this is by keeping the "box" class that assigns that specific element its shape and add an additional class to each box to specify the color of the box. For example:

```
<div class="boxes">
    <div class = "box box1">Box 1</div>
    <div class = "box box2">Box 2</div>
    <div class = "box box3">Box 3</div>
</div>
```

Although this method is essentially another way of writing what you're doing lines 45-54, we keep the styling located in the same file.